src/decompression_stream_ffi.mjs

```diff
@@ -0,0 +1,12 @@
+import { toBitArray } from "../gleam_stdlib/gleam.mjs"
+
+export const decompress = async (data, encoding) => {
+  const ds = new DecompressionStream(encoding);
+  const blob = new Blob([data.buffer])
+  const decompressedStream = blob.stream().pipeThrough(ds)
+
+  return await new Response(decompressedStream)
+        .blob()
+        .then(blob => blob.arrayBuffer())
```

Comment:
Can you not use await and then in the same context? I found it rather confusing.
My preference would be for all await style.

Reply:
done. I'm getting more and more familiar with the JS way of writint this ;)